src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -144,6 +147,16 @@ def __get_attendance_events() -> pd.DataFrame:
     return __get_attendance_events
 
 
+def _get_system_activities(input_directory: str) -> Callable:
+    def __get_system_activities() -> pd.DataFrame:
+        system_activities = analyticsFacade.get_system_activities(input_directory)
+        result_bucket["system_activities"] = system_activities
```

Comment:
I never see `result_bucket["system_activities"]` being used it again - is there a plan to access it, or perhaps can we remove that assignment altogether?

Reply:
I can remove it.

Reply:
✔️